### PR TITLE
Storybook additions

### PR DESCRIPTION
**Pull request by rrhyne**
Two small storybook additions text-muted and labels. I needed to see what they looked like for Figma component work.

### CODE REVIEW

**Comment by rrhyne** [comment]
Thanks, Erik, I inadvertently based this off my last branch. I should take this opportunity to learn to rebase but I'll do that on my time and start a new clean branch and PR to complete this.

File: web/src/global-styles/global-styles.story.tsx [hunk 1]
```diff
@@ -64,6 +64,14 @@ add('Text', () => (
             </small>
         </p>
 
+        <p>
+            <label>'Labels'</label> help label things
```
Comment (felixfbecker) [inline web/src/global-styles/global-styles.story.tsx:68]: I'm not sure if this is the intended use of `<label>` - we have them shown in the forms story, I've never seen them be used inline in text?

Comment (rrhyne) [reply web/src/global-styles/global-styles.story.tsx:68]: I think you are right. I saw them as part of the form group but was looking for a stand-alone version to easily compare styles with and to mirror the independent component in Figma for easier viewing.

**Comment by rrhyne** [comment]
Clean PR is here: https://github.com/sourcegraph/sourcegraph/pull/12418

**Pull request closed without merging**